src/experimental/react/controllers/animation-map-controller.js

```diff
@@ -83,8 +86,8 @@ const getDefaultCursor = ({isDragging}) => isDragging ? CURSOR.GRABBING : CURSOR
 
 const defaultProps = Object.assign({}, MAPBOX_LIMITS, {
   onViewportChange: null,
-  animateViewport: false,
   viewportAnimationDuration: VIEWPORT_ANIMATION_DURATION,
+  viewportAnimationFunc: viewportLinearAnimation,
```

Comment:
Probably for a follow-up PR: Can this be generalized? Does it need to be called `viewportLinearAnimation`, or can it just be `linearAnimation`?